roofit/roofitcore/inc/RooAbsPdf.h

```diff
@@ -285,7 +285,8 @@ class RooAbsPdf : public RooAbsReal {
   static void verboseEval(Int_t stat) ;
   static int verboseEval() ;
 
-  double extendedTerm(double sumEntries, const RooArgSet* nset, double sumEntriesW2=0.0) const;
```

Comment:
Is this an interface change for a public function?
We have to be careful with this, since there could be somebody using this function.

Reply:
I don't know why GitHub show this as a change. The old interface is still there, I just added an overload where you give the number of expected events directly

Reply:
OK, sometimes it is not easy to see in Github, thanks for the clarifications!